jetty-server/src/test/java/org/eclipse/jetty/server/session/SessionHandlerTest.java

```diff
@@ -0,0 +1,41 @@
+//
+//  ========================================================================
+//  Copyright (c) 1995-2019 Mort Bay Consulting Pty. Ltd.
+//  ------------------------------------------------------------------------
+//  All rights reserved. This program and the accompanying materials
+//  are made available under the terms of the Eclipse Public License v1.0
+//  and Apache License v2.0 which accompanies this distribution.
+//
+//      The Eclipse Public License is available at
+//      http://www.eclipse.org/legal/epl-v10.html
+//
+//      The Apache License v2.0 is available at
+//      http://www.opensource.org/licenses/apache2.0.php
+//
+//  You may elect to redistribute this code under either of these licenses.
+//  ========================================================================
+//
+
+package org.eclipse.jetty.server.session;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import javax.servlet.SessionTrackingMode;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashSet;
+
+public class SessionHandlerTest
+{
+    @Test
+    public void testSessionTrackingMode()
+    {
+        SessionHandler sessionHandler = new SessionHandler();
+        sessionHandler.setSessionTrackingModes(new HashSet<>(
+            Arrays.asList(SessionTrackingMode.COOKIE, SessionTrackingMode.URL)));
+        sessionHandler.setSessionTrackingModes(Collections.singleton(SessionTrackingMode.SSL));
```

Comment:
Formatting?

Reply:
ok I will change that but it's a test and I try to avoid to long lines as I use a laptop :)

Reply:
@olamy on, hang on. The formatting looks different depending on how I view it. From the "Files changed" tab, line 37 is all screwed up, but if I view the whole file, it's fine?!

Reply:
this part looks good. There were only issues with import order.